epictrack-web/src/components/workPlan/status/StatusView/StatusHistory/HistoryItem.tsx

```diff
@@ -68,7 +69,7 @@ const HistoryItem = ({ status }: HistoryItemProps) => {
           <TimelineContent>
             <Box>
               <ETStatusHistoryDate>
-                {moment(status.posted_date).format("L")}
+                {moment(status.posted_date).format("MMM.DD YYYY")}
```

Comment:
I think we will use this format in many other places. Shall we have this as a constant. We already have DATE_FORMAT as a constant. This can be another format.